smsdk/client.py

```diff
@@ -225,3 +225,51 @@ def get_machine_schema(self, machine_source, types=[], return_mtype=False, **kwa
                             f"Unknow stat schema identified :: machine_type {machine_source} - "
                             f"title_prefix :: {stat.get('display', {}).get('title_prefix', '')}")
         return fields
+    
+    def get_kpis(self, **kwargs):
+        kpis = smsdkentities.get('kpi')
+        base_url = get_url(
+            self.config["protocol"], self.tenant, self.config["site.domain"]
+        )
+        return kpis(self.session, base_url).get_kpis(**kwargs)
+    
+    def get_kpis_for_asset(self, **kwargs):
+        kpis = smsdkentities.get('kpi')
+        base_url = get_url(
+            self.config["protocol"], self.tenant, self.config["site.domain"]
+        )
+        return kpis(self.session, base_url).get_kpis_for_asset(**kwargs)
+    
+    def get_kpi_data_viz(self, machine_source=None, kpis=None, i_vars=None, time_selection=None, **kwargs):
```

Comment:
Let's have this take a `machines` array, not just a single machine, since that's more flexible, and the API supports it.